tests/utils.go

```diff
@@ -100,3 +105,29 @@ func GetKubeClientFromRESTConfig(config *rest.Config) (*kubernetes.Clientset, er
 
 	return coreClient, nil
 }
+
+// Creates a new namespace with a randomly generated name that starts with cdi-test-
+func GenerateNamespace() *k8sv1.Namespace {
+	client, err := GetKubeClient()
+	PanicOnError(err)
```

Comment:
I'm not sure the reason for calling Panic here vs Expect. I guess you want the entire suite to fail immediately.